pyquil/tests/data/compiler-isa-Aspen-8.json

```diff
@@ -1,1054 +1,851 @@
 {
-    "2Q": {
-        "0-1": {
-            "gates": [],
-            "ids": [
-                0,
-                1
-            ]
-        },
-        "0-7": {
+    "1Q": {
```

Comment:
Just curious, were the changes in this file due to handmade or generated changes?

Reply:
Generated. I definitely didn't hand write this, but rather generated the CompilerISA and dumped it to file, then eye-balled.